pandas_summary/__init__.py

```diff
@@ -20,11 +20,12 @@ class DataFrameSummary(object):
     TYPE_CONSTANT = 'constant'
     TYPE_UNIQUE = 'unique'
 
-    def __init__(self, df):
```

Comment:
I think we can make it default to False instead, most users are only interested in the numerical values.

Reply:
Done!